charts/cronjob-gitlab-delete-projects/README.md

```diff
@@ -7,10 +7,24 @@ A cronjob that enables OpenShift to delete Gitlab projects that have become stal
 The cronjob will not delete in the follow cases:
 
 1. A group with the text `DO_NOT_DELETE` in the description will not delete the group, any subgroup and projects in the group and subgroups.
-2.  A project wit the text `DO_NOT_DELETE` in the description or a tag `DO_NOT_DELETE` in the tag list.
+2. A project wit the text `DO_NOT_DELETE` in the description or a tag `DO_NOT_DELETE` in the tag list.
 
 **Caution:** this jobs performs hard deletes. The actual deletion policy is influenced by overarching settings in Gitlab.
 
+## Notification / Hook
+
+This job has the ability to notify applications via a webhook. Specify the url to post to and the value of the secret header `x-cleanup-token` for the request to be invoked.
```

Comment:
Looks like `x-cleanup-token` here but `x-notification-token` in the implementation in `gitlab-cleanup.py`? Otherwise lgtm